SixTrack/sixtrack.s

```diff
@@ -12274,6 +12308,8 @@ cc2008
       if(idat.eq.dynk)  goto 2200
       if(idat.eq.fma)   goto 2300
       if(idat.eq.elens) goto 2400
+      if(idat.eq.dist)  goto 2500
```

Comment:
Wire (Pull request #174, likely to be merged before DIST) already uses `goto 2500`, could you use 2600 and 2700?